### PR TITLE
Document PackageDownload

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2404

Reference design doc: https://github.com/NuGet/Home/wiki/%5BSpec%5D-PackageDownload-support

### CODE REVIEW

File: docs/consume-packages/PackageDownload-Functionality.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+# PackageDownload
+
+Starting with Visual Studio 2017 and .NET SDK 1.0.0, NuGet [PackageReference](Package-References-in-Project-Files.md) functionality was added.
+
+`PackageReference` allows you to manage your package dependencies directly in your project file.
+When you run restore, the transitive dependencies are resolved automatically and the applicable references are chosen for each package in the project graph.
+
+In [5.3 of NuGet](..\release-notes\NuGet-5.3.md) a companion functionality was introduced for [.NET SDK-style projects](..\resources\check-project-format.md) called `PackageDownload`.
```
Comment (zivkan) [inline docs/consume-packages/PackageDownload-Functionality.md:17]:
```suggestion
In [NuGet version 5.3](..\release-notes\NuGet-5.3.md) a companion feature was introduced for [.NET SDK-style projects](..\resources\check-project-format.md) called `PackageDownload`.

```

File: docs/consume-packages/PackageDownload-Functionality.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+|------------|-------------|---------|
+| Version | Only exact versions, surrounded with `[]` are supported. Multiple versions can be specified separated by `;` | `[1.0.0]`, `[1.0.0];[2.0.0]` |
+
+Packages acquired through PackageDownload will undergo the same [installation process](..\concepts\package-installation-process.md) as packages acquired through PackageReference.
+This means [package signatures](installing-signed-packages.md) are validated, [package source mapping](Package-Source-Mapping.md) is considered.
+All newly acquired PackageDownload packages will be installed in the global packages folder.
+
+| Feature | PackageReference | PackageDownload |
+|-|------------------|-----------------|
+| Package assets selection | Assemblies from packages are automatically added to the project and can be used for compile and runtime | Assemblies or any other assets from the package are not included in the project. |
```
Comment (zivkan) [inline docs/consume-packages/PackageDownload-Functionality.md:35]:
```suggestion
| Package assets selection | Assemblies from packages are automatically added to the project and can be used for compile and runtime | No assets from the package are included in the project. |

```

File: docs/consume-packages/PackageDownload-Functionality.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+| dotnet list package | All dependencies are included | PackageDownload packages are not shown by `dotnet list package`. |
+
+Due to the fact that PackageDownload are not tied to the project in any way beyond acquisition, multiple versions of the same package can be downloaded.
+
+### PackageDownload limitations
+
+Given that this is an advanced feature with limited applicability, it doesn't have a tooling support equivalent to PackageReference.
+
+- There is no VisualStudio or dotnet.exe functionality to modify PackageDownload items. You can only change them manually in your project files.
+- dotnet add, remove, and list commands do not account for PackageDownload items.
```
Comment (zivkan) [inline docs/consume-packages/PackageDownload-Functionality.md:49]: Do lock file take PackageDownload into consideration?  If not, this would be an important limitation to list considering some customers use lock files & content hash validation as a layer of supply chain security.

**Review by zivkan — APPROVED** [review]

File: docs/consume-packages/PackageDownload-Functionality.md (hunk 3 above)
Comment (nkolev92) [reply docs/consume-packages/PackageDownload-Functionality.md:51]: No, they do not. 

That's a great point and something I should have added.

File: docs/consume-packages/PackageDownload-Functionality.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+# PackageDownload
+
+Starting with Visual Studio 2017 and .NET SDK 1.0.0, NuGet [PackageReference](Package-References-in-Project-Files.md) functionality was added.
+
+`PackageReference` allows you to manage your package dependencies directly in your project file.
+When you run restore, the transitive dependencies are resolved automatically and the applicable references are chosen for each package in the project graph.
+
+In [NuGet version 5.3](..\release-notes\NuGet-5.3.md) a companion feature was introduced for [.NET SDK-style projects](..\resources\check-project-format.md) called `PackageDownload`.
```
Comment (JonDouglas) [inline docs/consume-packages/PackageDownload-Functionality.md:17]: What does the feature do in summary? Would add it after the `PackageDownload` part here. 

i.e. Packages acquired through PackageDownload will undergo the same installation process as packages acquired through PackageReference

File: docs/consume-packages/PackageDownload-Functionality.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+---
+title: PackageDownload
```
Comment (JonDouglas) [inline docs/consume-packages/PackageDownload-Functionality.md:2]: Download packages with `PackageDownload`?

**Review by JonDouglas — APPROVED** [review]
Left some comments to consider. Generally LGTM.

File: docs/consume-packages/PackageDownload-Functionality.md (hunk 4 above)
Comment (nkolev92) [reply docs/consume-packages/PackageDownload-Functionality.md]: I did my best to add a one liner. It's pretty hard to describe this feature in one liner. 

tldr is that it downloads a specific version, without its dependencies and without any assets selected. 

Unfortunately that'd be a lot of big words that for which it's not super obvious what they mean, so I tried to simplify it as much as I can.

File: docs/consume-packages/PackageDownload-Functionality.md (hunk 5 above)
Comment (nkolev92) [reply docs/consume-packages/PackageDownload-Functionality.md:2]: Sure. Changed.

File: docs/consume-packages/PackageDownload-Functionality.md (hunk 4 above)
Comment (nkolev92) [reply docs/consume-packages/PackageDownload-Functionality.md]: I see you already approved. I'd appreciate a quick look at the addition in this paragraph @JonDouglas

File: docs/consume-packages/PackageDownload-Functionality.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+Ideally, all your dependencies are expressed through PackageReference, but in scenarios where that's not possible, or often times not practical yet, you can use this feature to simply `download` packages to a certain location, in a similar way that you could achieve that with a `packages.config` file not tied to a project. 
+
+Example:
+
+```xml
+
+<Project Sdk="Microsoft.Build.NoTargets/1.0.80"> <!-- This is not a project we want to build. -->
+
+    <PropertyGroup>
+        <RestorePackagesPath>packages/</RestorePackagesPath> <!-- Changes the global packages folder-->
```
Comment (erdembayar) [inline docs/consume-packages/PackageDownload-Functionality.md:65]: How different this `RestorePackagesPath` from `globalPackagesFolder`?

Comment (nkolev92) [reply docs/consume-packages/PackageDownload-Functionality.md:65]: It's a different way to achieve the same thing.

This is in a project file, hence self contained.

File: docs/consume-packages/PackageDownload-Functionality.md (hunk 4 above)
Comment (JonDouglas) [reply docs/consume-packages/PackageDownload-Functionality.md]: looks great! consider this closed.
